cfme/tests/cloud/test_delete_cloud_object.py

```diff
@@ -48,7 +47,8 @@ def test_delete_instance(setup_provider, provider):
     """
     instance_name = provider.data['remove_test']['instance']
     test_instance = VM.factory(instance_name, provider)
-    test_instance.delete()
+    delete_text = 'Remove selected item'
```

Comment:
Okay - common.vm.delete() uses a constant for the cfg_btn text, which will need to be version picked. Additionally the cfg_btn now is more specific, and has "Remove Instance" on the details page. I'll see what I can come up with.